src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -120,24 +120,24 @@ public String getApiUri() {
     @Override public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {
         TaskListener listener = observer.getListener();
         if (repoOwner.isEmpty()) {
-            listener.getLogger().format("Must specify user or organization%n");
-            return;
+            throw new AbortException("Must specify user or organization%n");
```

Comment:
:bug: this is actually bad, as it will cause indexing to break on an as-yet-unconfigured folder. Revert.